src/patchers/UnaryExistsOpPatcher.js

```diff
@@ -0,0 +1,24 @@
+import UnaryOpPatcher from './UnaryOpPatcher.js';
+
+export default class UnaryExistsOpPatcher extends UnaryOpPatcher {
+  patchAsExpression() {
+    const node = this.node;
+    const nodeExpression = node.expression;
+
+    // Only add parentheses if necessary
+    if (node.parentNode && node.parentNode.type !== 'Block') {
```

Comment:
This is pretty much trying to replicate the statement vs. expression thing. What I'd probably do is to put everything from `patchAsExpression` into `patchAsStatement` and then call the latter from the former like so:

```js
patchAsExpression() {
  this.insertBefore('(');
  this.patchAsStatement();
  this.insertAfter(')');
}
```